Python/pytime.c

```diff
@@ -779,19 +779,80 @@ _PyTime_GetMonotonicClockWithInfo(_PyTime_t *tp, _Py_clock_info_t *info)
     return pymonotonic(tp, info, 1);
 }
 
+
+#ifdef MS_WINDOWS
 int
-_PyTime_Init(void)
+_PyTime_GetWinPerfCounterWithInfo(_PyTime_t *t, _Py_clock_info_t *info)
```

Comment:
And this accepted a `double *t` as the output parameter?

Reply:
+1